src/components/Doctors/DeleteDoctor.js

```diff
@@ -1,7 +1,59 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+// import { useNavigate } from 'react-router-dom';
+// import localStorages from '../../helpers/localStorage';
+import {
+  deleteDoctorAction,
+  getDoctorsAction,
+} from '../../redux/doctorReduce/doctors';
+import './delete_doctor.css';
 
-const DeleteDoctor = () => (
-  <div>DeleteDoctor</div>
-);
+const DeleteDoctor = () => {
+  const doctors = useSelector((state) => state.doctor);
+  const [successNotice, setSuccessNotice] = useState(false);
+  // const navigate = useNavigate();
+  // const hasAccount = localStorages.getUser();
+  // const role = hasAccount.user === undefined ? '' : hasAccount.user.name;
+
+  // useEffect(() => {
+  //   if (role !== 'admin') {
+  //     navigate('/401');
+  //   }
+  // }, []);
```

Comment:
- [Optional] Please kindly remove commented code from the component to look clean and more readable